packages/ng/style/definitions/select/_select-clearer.scss

```diff
@@ -63,4 +55,24 @@
 	:host-context(.textfield-input.is-disabled, .textfield-input[disabled]) {
 		display: none;
 	}
+
+	// PALETTES
+	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+	@each $name, $palette in _getMap("palettes") {
+		:host-context(.palette-#{$name}) {
+			.actionIcon {
+				@include selectColoringClearer($name);
+			}
+		}
+	}
+}
+
+@mixin selectColoringClearer($palette) {
+		color: _color($palette, 700);
+
+		&:hover {
```

Comment:
No focus management too?

Reply:
Yes, it should be a focusable element, to be easily reached with keyboard and mouse.

Even if it's not the case yet, I think we can plan the CSS style accordingly.